hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusGetTopicInfoHandler.java

```diff
@@ -134,21 +136,20 @@ public Response findResponse(@NonNull final QueryContext context, @NonNull final
      */
     private Optional<ConsensusTopicInfo> infoForTopic(
             @NonNull final TopicID topicID, @NonNull final ReadableTopicStore topicStore) {
-        final var meta = topicStore.getTopicMetadata(topicID);
+        final var meta = topicStore.getTopic(topicID);
         if (meta == null) {
             return Optional.empty();
         } else {
             final var info = ConsensusTopicInfo.newBuilder();
-            meta.memo().ifPresent(info::memo);
-            info.runningHash(Bytes.wrap(meta.runningHash()));
+            info.memo(meta.memo());
+            info.runningHash(meta.runningHash());
             info.sequenceNumber(meta.sequenceNumber());
-            info.expirationTime(meta.expirationTimestamp());
+            info.expirationTime(Timestamp.newBuilder().seconds(meta.expiry()).build());
             if (!isEmpty(meta.adminKey())) info.adminKey(meta.adminKey());
             if (!isEmpty(meta.submitKey())) info.submitKey(meta.submitKey());
-            info.autoRenewPeriod(Duration.newBuilder().seconds(meta.autoRenewDurationSeconds()));
-            meta.autoRenewAccountId()
-                    .ifPresent(account ->
-                            info.autoRenewAccount(AccountID.newBuilder().accountNum(account)));
+            info.autoRenewPeriod(Duration.newBuilder().seconds(meta.autoRenewPeriod()));
+            if (meta.autoRenewAccountNumber() != 0)
```

Comment:
@Neeharika-Sompalli or @tinker-michaelj should this be
```suggestion
            if (meta.autoRenewAccountNumber() > 0)
```
?

Reply:
The builder default autoRenewAccountNumber is 0. I will also let @Neeharika-Sompalli and @tinker-michaelj answer this.

Reply:
Since we validate the auto-renew account at the time it is set on the `Topic`, the only options here will be:
 1. `number = 0` (no auto-renew account); or,
 2. `number = X > 0` (where `0.0.X` was a valid auto-renew account at the time it was set on this `Topic`).

If you want, we could log at `ERROR` if the number is negative; but it should be impossible.

Reply:
Ah got it, ok that works. Thanks!